src/Kunstmaan/GeneratorBundle/Command/InstallCommand.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace Kunstmaan\GeneratorBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputDefinition;
+use Symfony\Component\Console\Output\OutputInterface;
+use Sensio\Bundle\GeneratorBundle\Command\Validators;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Process\Exception\RuntimeException;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Sensio\Bundle\GeneratorBundle\Command\Helper\QuestionHelper;
+use Symfony\Component\Debug\Exception\ContextErrorException;
+
+/**
+ * Kunstmaan installer
+ */
+class InstallCommand extends Command
+{
+    /**
+     * @var int
+     */
+    private $commandSteps = 0;
+
+    /**
+     * @var ContainerInterface
+     */
+    private $containter;
+
+    /**
+     * @param ContainerInterface $container Container
+     */
+    public function __construct(ContainerInterface $container)
+    {
+        $this->container = $container;
```

Comment:
Symfony 4 is deprecating using the container directly, in the bundles we also have several attempts going on to remove all places where we are injecting and using the database. I do not think its a good idea to add new code which uses the container now. Better just inject the dependencies you want.